### DISCUSSION TITLE
External links syntax

### DISCUSSION THREAD

**Discussion by chuckis** [markqvist/Reticulum#34] [Help & Questions] [answered]
Hi! I wonder how to insert external links with micron syntax.
I tried to follow the guide and did like this `\`F00a`_`[here`https://t.me/blah]`_`f\`
But it says: " Could not open link: Malformed URL "

**Comment by 4c3e** [comment, marked as answer]
Do you mind sharing the context of the error? Was it thrown when you tried to click on that link using NomadNet? Or was it thrown when compiling the micron document. 

If it was the first case, it's because NomadNet doesn't support loading resources over https. I suppose NomadNet could better support this by alerting the user about a link to an out-of-reticulum resource.

**Comment by markqvist** [comment]
It looks like the error that NomadNet throws when a user tries to open an invalid link, which it would be since, as @4c3e correctly mentions, NomadNet does not support links to external resources currently.

The best way for you to include external links right now would be to simply include the URL in text, and maybe underline it in micron, to visiually signify that it is an URL. Most terminals will autodetect URLs and let you click and open in an external browser.

We could add suppor for external links easily though. Maybe it makes sense? I am a bit hesistant to enable it by default though, at least until security implications have been thought through.

But either way, it should be simple to add. As far as I remember, Python has a native way of calling the external URL handler of the operating system, so NomadNet would just need to detect http/https links and discern those from normal page links.
